NDTensors/src/lib/SmallVectors/test/runtests.jl

```diff
@@ -1,5 +1,6 @@
+@eval module $(gensym())
```

Comment:
This seems to fail with `f!_impl_broken` not defined

Reply:
Probably an issue with having nested `@eval`, maybe you could try removing the inner one. If that doesn't work, you can just remove this module wrapper and we can figure it out later, not a priority for this PR.

Reply:
I tried removing the inner `eval` and that didn't work so I removed it from the file for now